pkg/updater/updater.go

```diff
@@ -285,46 +312,24 @@ func Update(parent context.Context, client gcs.ConditionalClient, mets *Metrics,
 			log.Debug("Started fixers on current test groups")
 		}
 		fixAll()
-		cond := storage.Conditions{GenerationNotMatch: cfgAttrs.Generation}
-		opener := client.If(&cond, &cond)
-		ticker := time.NewTicker(time.Minute) // TODO(fejta): subscribe to notifications
```

Comment:
Why pull this out? It seems fine to me to continue doing both the occasional logging and the occasional cfg snapshot update inside the same go routine?

Reply:
It seemed clearer to have each goroutine doing one thing; one for logging and one for config updating. I'm fine with moving it back.

Reply:
Fair enough, either way is good with me.